src/core/Memory.ts

```diff
@@ -1,9 +1,16 @@
-let values_in_browser: { date: Date, value: number }[] = [];
+export const storageKey = 'values';
+
+function parseValueDateToBeDateObj (valuesFromLS: { date: string, value: number }[]): { date: Date, value: number }[] {
+    return valuesFromLS.map(dataObj => ({
```

Comment:
I come from Java environment where we are not scared of long names.
Here using valuesFromLocalStorage sound better.

Also I would name your function like recoverTypesFromParsedJson to make it more understandable what is it doing.

Reply:
:) agree